scripts/doc_checker.py

```diff
@@ -201,6 +232,8 @@ def validate_modin_error(doc: Docstring, results: dict) -> list:
     """
     errors = check_optional_args(doc)
     errors += check_spelling_words(doc)
+    if any((error_code == "PR01" for error_code, _ in results["errors"])):
+        errors += check_parameters_indention(doc)
```

Comment:
```suggestion
    errors += check_parameters_indention(doc)
```
we should check indentation regardless of other checks